fossunited/www/newsletter/index.html

```diff
@@ -1,176 +1,560 @@
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: auto;
+    align-items: center;
+    gap: 2.5rem;
+    height: auto;
+    background: #f5f5f5;
+  }
+
+  .text-content {
+    text-align: center;
+  }
+
+  .text-content h1 {
+    color: hsl(var(--clr-code-night-900));
+    text-align: center;
+    font-family: Inter;
+    font-size: var(--text-4xl);
+    font-style: normal;
+    font-weight: 700;
+    line-height: 2.75rem;
+    /* 110% */
+    letter-spacing: -0.1rem;
+  }
+
+  .subscribe-section {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  #email {
+    display: flex;
+    width: 25rem;
+    height: 2.5rem;
+    padding: 0.6875rem 0.75rem;
+    align-items: center;
+    gap: 0.875rem;
+    border-radius: 0.5rem;
+    border: hsl(var(--clr-gray-900));
+    background: hsl(var(--clr-gray-50));
+  }
+
+  #subscribe-button {
+    display: flex;
+    width: 9.125rem;
+    height: 2.5rem;
+    padding: 0.625rem 0.75rem;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    gap: 0.5rem;
+    border-radius: 0.5rem;
+    background: hsl(var(--clr-foss-mint-500));
+  }
+
+  .news-icon img {
+    display: flex;
+    width: 28rem;
+    height: 22rem;
+
+    justify-content: center;
+    align-items: center;
+  }
+
+  #past-news {
+    margin: auto;
+    width: auto;
+  }
+
+  .newsletter-list {
+    display: inline-flex;
+    padding: 2.5rem;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 2.5rem;
+    width: 100%;
+    margin: auto;
+  }
+
+  .newsletter-list h4 {
+    width: 100%;
+  }
+
+  .news {
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: row;
+    width: auto;
+  }
+
+  #news-details {
+    width: 320px;
+    height: auto;
+    gap: 12px;
+    padding: 1rem 1rem;
+    margin: 15px;
+    border-radius: 8px;
+    flex-wrap: wrap;
+    border: 0.5px solid hsl(var(--clr-gray-400));
+  }
+
+  #news-details h6 {
+    height: 45px;
+    font-size: var(--text-lg);
+    font-style: normal;
+    font-weight: 600;
+    line-height: 100%;
+    /* 21.6px */
+  }
+
+  #news-details a {
+    color: hsl(var(--clr-foss-mint-500));
+    font-size: var(--text-sm);
+    line-height: 12px;
+    margin-bottom: 2rem;
+  }
+
+  .news-icon {
+    display: flex;
+    margin: auto;
+  }
+
+  #success {
+    margin: auto;
+    width: 500px;
+  }
+
+  #success-icon-div {
+    height: 200px;
+    width: 500px;
+  }
+
+  .success-icon {
+    margin-bottom: -60px !important;
+    width: 100px !important;
+    height: 250px !important;
+  }
+
+  div#success-wrapper {
+    display: flex;
+    height: 100vh;
+    width: 500px;
+    margin: auto;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  #subscribed-page {
+    display: none;
+  }
+
+  .subscribed-page-show {
+    display: block;
+  }
+
+  #footer {
+    background: hsl(var(--clr-gray-100));
+    padding: 1.5rem 1.5rem;
+  }
```

Comment:
footer? we already have a footer. If this is different then use better id names and class-names